feat(ecs-patterns): cooldown parameter to QueueProcessingServiceBaseProps (#28730)
aws/aws-cdk: packages/aws-cdk-lib/aws-ecs-patterns/README.md

});
```

> ### Set Cooldown parameter for QueueProcessingFargateService

Reviewer: Missing one sentence explaining what the cooldown parameter is

Author: added sentence below, let me know if it needs changing

```ts
declare const vpc: ec2.Vpc;
const queueProcessingFargateService = new ecsPatterns.QueueProcessingFargateService(this, 'Service', {
  vpc,
  memoryLimitMiB: 512,
  image: ecs.ContainerImage.fromRegistry('test'),
  assignPublicIp: true,

>   cooldown: Duration.seconds(500)

Reviewer: ```suggestion
  cooldown: Duration.seconds(500),
```

});
```

> ### Set Cooldown parameter for QueueProcessingFargateService

Reviewer: ```suggestion
### Set cooldown for QueueProcessingFargateService
```

> ### This will specify the amount of time, in seconds, to wait for a previous scaling activity to take effect.

Reviewer: ```suggestion

The cooldown period is the amount of time to wait for a previous scaling activity to take effect.
To specify something other than the default cooldown period of 300 seconds, use the `cooldown` parameter: 
```
